eng/convert-codecoveragetoxml.ps1

```diff
@@ -0,0 +1,21 @@
+[cmdletbinding()]
```

Comment:
I am quite surprised we need to do this and there's no automatic way (or an AzDO task) for this. Can you please check? You can also ask the VSTest team (Jakub Chocholowiz or Marco Rossignoli for instance) who might have some tips

Reply:
We might not to be honest, I already removed a few things that were not needed, maybe this is one of them too, I'll try removing it first

Reply:
Great, I believe we should be removing bloat that maybe was needed some time ago but is not anymore. If it's too much work, just dont worry about it now but looks fishy

Reply:
Looks like it worked without the attribute.

Reply:
Wait, I meant the whole script 😅

Reply:
And the associated YAML build step and so on

Reply:
Ahh ahah, that's a different story..

Reply:
Yup, these shenanigans were needed when I created this stuff like... 3-4 years ago. If there's an easier way to do it, let's do it. If it's too much effort to invest in right now, let's make an issue to revisit it later. (It was a pain in the ass and I spent a long time figuring out how to get this stuff to work :D )